openedx_tagging/core/tagging/rest_api/v1/views.py

```diff
@@ -488,22 +508,24 @@ def update(self, request, *args, **kwargs) -> Response:
         taxonomy = query_params.validated_data.get("taxonomy", None)
         taxonomy = taxonomy.cast()
 
-        perm = "oel_tagging.change_objecttag"
-
         object_id = kwargs.pop('object_id')
         perm_obj = ObjectTagPermissionItem(
             taxonomy=taxonomy,
             object_id=object_id,
         )
-
-        if not request.user.has_perm(
-            perm,
-            # The obj arg expects a model, but we are passing an object
-            perm_obj,  # type: ignore[arg-type]
-        ):
-            raise PermissionDenied(
-                "You do not have permission to change object tags for this taxonomy or object_id."
-            )
+        perms = [
+            "oel_tagging.add_objecttag",
+            "oel_tagging.delete_objecttag"
+        ]
```

Comment:
I would prefer if we create an `oel_tagging.can_tag_object` to replace `add_objecttag`, `delete_objecttag`and `change_objecttag`, but I'm fine with this approach.

Reply:
I agree. `add_objecttag` is ambiguous, because does it mean you can tag a specific object or use a specific taxonomy to tag any object, or you can add object tags in general but maybe not in particular?

`can_tag_object` is very clear what it means - you can tag that object. (add/change/remove tags)

Reply:
Makes sense to me too, thank you for pushing for consensus here, @rpenido !

See 3dfea40